packages/cfd/src/features/Containers/cfd-password-manager-modal.tsx

```diff
@@ -194,20 +203,46 @@ const CFDPasswordManagerTabContent = ({
         setIsSubmitSuccessInvestor(true);
     };
 
+    React.useEffect(() => {
+        if (change_password_status === QUERY_STATUS.ERROR && change_password_error) {
+            showError((change_password_error as unknown as Error)?.message);
+        }
+        if (change_password_status === QUERY_STATUS.SUCCESS) {
+            hideError();
+        }
+    }, [change_password_error, change_password_status]);
+
+    React.useEffect(() => {
+        if (change_investor_password_status === QUERY_STATUS.ERROR && change_investor_password_error) {
+            showError((change_investor_password_error as unknown as Error)?.message);
+        }
+        if (change_investor_password_status === QUERY_STATUS.SUCCESS) {
+            hideError();
+        }
+    }, [change_investor_password_error, change_investor_password_status]);
+
```

Comment:
Just a suggestion. Maybe you can create a helper function for the status like so: 

`const handleStatus = (status: QUERY_STATUS, error: any) => {
    if (status === QUERY_STATUS.ERROR && error) {
        showError((error as unknown as Error)?.message);
    }
    if (status === QUERY_STATUS.SUCCESS) {
        hideError();
    }
};`

```suggestion
   React.useEffect(() => {
    handleStatus(change_password_status, change_password_error);
}, [change_password_error, change_password_status]);

React.useEffect(() => {
    handleStatus(change_investor_password_status, change_investor_password_error);
}, [change_investor_password_error, change_investor_password_status]);
```

Reply:
If i do so then i have to make a callback for that function which will also call unnecessary renders. 🤔

Reply:
Do it then 👿